File: Example/GBFlatButtonExample/ViewController.m
//
//  ViewController.m
//  GBFlatButtonExample
//
//  Created by Gustavo Barbosa on 4/15/14.
//  Copyright (c) 2014 Gustavo Barbosa. All rights reserved.
//

#import "ViewController.h"
#import "GBFlatButton.h"
#import "UIColor+GBFlatButton.h"

@interface ViewController ()

@property (nonatomic, weak) IBOutlet GBFlatButton *pinkButton;
@property (nonatomic, weak) IBOutlet GBFlatButton *yellowButton;
@property (nonatomic, weak) IBOutlet GBFlatButton *orangeButton;
@property (nonatomic, weak) IBOutlet GBFlatButton *greenButton;
@property (nonatomic, weak) IBOutlet GBFlatButton *blueButton;
@property (nonatomic, weak) IBOutlet GBFlatButton *purpleButton;

@property (nonatomic, weak) IBOutlet GBFlatButton *pinkSelectedButton;
@property (nonatomic, weak) IBOutlet GBFlatButton *yellowSelectedButton;
@property (nonatomic, weak) IBOutlet GBFlatButton *orangeSelectedButton;
@property (nonatomic, weak) IBOutlet GBFlatButton *greenSelectedButton;
@property (nonatomic, weak) IBOutlet GBFlatButton *blueSelectedButton;
@property (nonatomic, weak) IBOutlet GBFlatButton *purpleSelectedButton;

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];

    // Uncomment the following lines to add a programatically-created button
//    GBFlatSelectableButton *blackButton = [[GBFlatSelectableButton alloc] initWithFrame:CGRectMake(0, 30, 150, 44)];
//    blackButton.tintColor = [UIColor blackColor];
//    [blackButton setTitle:@"Black"
//                 forState:UIControlStateNormal];
//    blackButton.center = CGPointMake(self.view.center.x, blackButton.center.y);
//    [self.view addSubview:blackButton];
    
    _pinkSelectedButton.selected = YES;
    _yellowSelectedButton.selected = YES;
    _orangeSelectedButton.selected = YES;
    _greenSelectedButton.selected = YES;
    _blueSelectedButton.selected = YES;
    _purpleSelectedButton.selected = YES;
}

@end

File: Example/GBFlatButtonSwiftExample/GBFlatButtonSwiftExample/ViewController.swift
//
//  ViewController.swift
//  GBFlatButtonSwiftExample
//
//  Created by Gustavo Barbosa on 6/10/14.
//  Copyright (c) 2014 Gustavo Barbosa. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    @IBOutlet var pinkButton: GBFlatButton?
    @IBOutlet var yellowButton: GBFlatButton?
    @IBOutlet var orangeButton: GBFlatButton?
    @IBOutlet var greenButton: GBFlatButton?
    @IBOutlet var blueButton: GBFlatButton?
    @IBOutlet var purpleButton: GBFlatButton?
    
    @IBOutlet var selectedPinkButton: GBFlatButton?
    @IBOutlet var selectedYellowButton: GBFlatButton?
    @IBOutlet var selectedOrangeButton: GBFlatButton?
    @IBOutlet var selectedGreenButton: GBFlatButton?
    @IBOutlet var selectedBlueButton: GBFlatButton?
    @IBOutlet var selectedPurpleButton: GBFlatButton?
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        var blackButton = GBFlatSelectableButton(frame: CGRect(x: 0, y: 0, width: 80, height: 40))
        blackButton.tintColor = UIColor.blackColor()
        blackButton.setTitle("black", forState: UIControlState.Normal)
        blackButton.center = self.view.center
        blackButton.frame.origin.y -= 30
        self.view.addSubview(blackButton)
        
        var redButton = GBFlatSelectableButton(frame: CGRect(x: 0, y: 0, width: 80, height: 40))
        redButton.tintColor = UIColor.redColor()
        redButton.setTitle("red", forState: UIControlState.Normal)
        redButton.center = self.view.center
        redButton.frame.origin.y += 30
        self.view.addSubview(redButton)

        selectedPinkButton!.selected = true
        selectedYellowButton!.selected = true
        selectedOrangeButton!.selected = true
        selectedGreenButton!.selected = true
        selectedBlueButton!.selected = true
        selectedPurpleButton!.selected = true
    }
}
